x/supplier/types/message_create_claim_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
```

Comment:
```suggestion
	TODO(@bryanchriswhite): Add unit tests for message validation when adding the business logic.
```

See #59 for an example